io/shared/src/main/scala/fs2/io/process/ProcessBuilder.scala

```diff
@@ -67,17 +70,34 @@ sealed abstract class ProcessBuilder private {
   /** @see [[workingDirectory]] */
   def withCurrentWorkingDirectory: ProcessBuilder
 
+  /** @see [[outputMode]] */
+  def withOutputConfig(outputConfig: ProcessOutputConfig): ProcessBuilder
```

Comment:
I think separating it out into three config methods for stdin/stdout/stderr (like the JDK process builder) will make the API easier to use.